packages/react-badge/src/BadgeIcon.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Badge } from '@fluentui/react-badge';
+import { Badge } from './index';
 import { AcceptIcon } from '@fluentui/react-icons-mdl2';
```

Comment:
you'll need to get rid of non v9 icons -> check how is it being done in react-menu for instance https://github.com/microsoft/fluentui/blob/master/packages/react-menu/src/MenuList.stories.tsx#L17

Reply:
Thank you for the help ^_^